daft/context.py

```diff
@@ -54,12 +54,22 @@ def _get_runner_config_from_env() -> _RunnerConfig:
 _RUNNER: Runner | None = None
 
 
+def _get_planner_from_env() -> bool:
+    """Returns whether or not to use the new query planner."""
+    rust_planner_env = os.getenv("DAFT_DEVELOPER_RUST_QUERY_PLANNER")
+    rust_planner = bool(int(rust_planner_env)) if rust_planner_env is not None else None
+
+    default = False
+    return rust_planner or default
```

Comment:
Nit: Can condense to a single, readable line using this pattern (we used this for Ray quite a bit, e.g. https://github.com/ray-project/ray/blob/b67258a86dbdca329d6a2a6d0756233130594909/python/ray/data/context.py#L83-L85).
```suggestion
    return bool(int(os.getenv("DAFT_DEVELOPER_RUST_QUERY_PLANNER", default="1")))
```